### ISSUE TITLE
Outdated information about OS X terminal.app

### ISSUE THREAD

**Issue by digiwano** [aam1r/sachet#9]
"Note: Some themes require a 256-color terminal to be used. If you are using Mac, iTerm is a great terminal that will suit your needs"

Since Lion, Terminal.app supports 256-color terms and even has its default TERM setting as xterm-256color.

**Comment by jeroenseegers** [comment]
I don't think this is meant as "Terminal.app doesn't support 256 colors", but more that iTerm actually is a much better terminal emulator than the native Terminal.app

**Comment by cweagans** [comment]
Well, it _was_. Now the Terminal app in OS X is pretty decent. If I wasn't switching back to Linux, I'd definitely use the Mac terminal.app
